src/scripts/main.js

```diff
@@ -1,8 +1,22 @@
 'use strict';
 
-const people = require('./lib/people');
+const people = require('./lib/people.json');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(human => {
+  const { sex, born, died, name: nameHuman } = human;
+  const gender = (personSex) => personSex === 'm' ? 'Male' : 'Female';
```

Comment:
Basically, you don't need a function here 
```suggestion
  const gender =  sex === 'm' ? 'Male' : 'Female';
```